channels/co.m3u

```diff
@@ -20,7 +20,7 @@ https://mdstrm.com/live-stream-playlist_800/58dc3d471cbe05ff3c8e463e.m3u8
 #EXTINF:-1 tvg-id="CNCPasto.co" tvg-name="CNC Pasto" tvg-country="CO" tvg-language="Spanish" tvg-logo="https://i.imgur.com/Hjc7XaY.png" group-title="",CNC Pasto (540p)
 http://66.240.236.25:1936/cncpasto/cncpasto/playlist.m3u8
 #EXTINF:-1 tvg-id="LATAMTV.tv" tvg-name="LATAM TV" tvg-country="CO" tvg-language="Spanish;English" tvg-logo="https://t46.90f.myftpupload.com/wp-content/uploads/2020/08/cropped-Logo-LATAM-Horizontal-solo-alfa.png" group-title="",LATAM TV (Bogotà)
-https://api.new.livestream.com/accounts/29967371/events/9247730/broadcasts/220706049.secure.m3u8
+https://iptv-all.lanesh4d0w.repl.co/latam
```

Comment:
Can you share the source code for this scraper? I'm trying to automate a few expiring links. Thanks

Reply:
Sure ! Pretty simple code, there you go

`const express = require("express");
const fetch = require('node-fetch');
const app = express();


app.get("/latam", function (req, res) {
  const urlTest = "https://player-api.new.livestream.com/accounts/29967371/events/9247730/stream_info"

  res.setHeader('Content-Type', 'application/vnd.apple.mpegurl;');

  fetch(urlTest)
    .then(response => response.json())
    .then(json => {
      res.redirect(json.m3u8_url)
    })
});

app.listen("8080");
module.exports = app;`